supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

```diff
@@ -191,6 +211,37 @@ Do not use "BIOS" as a generic term to refer to computer firmware. Use "firmware
 *See also*:
 
 // RHEL: Added "In Red Hat Enterprise Linux,"
+
+[[block-device]]
+==== image:images/caution.png[with caution] block device (noun)
+*Description*: A physical storage device that supports reading and writing data in groups (blocks) of bytes, in any order. Do not confuse for "block storage" (cloud storage) or "block volume" (OpenShift Virtualization).
```

Comment:
```suggestion
*Description*: A _block device_ is a physical storage device that supports reading and writing data in groups (blocks) of bytes in any order. Do not confuse this term with _block storage_ (cloud storage) or _block volume_ (OpenShift Virtualization).
```
Per ISG: Words and phrases that are being defined or explained should use italic text: https://www.ibm.com/docs/en/ibm-style?topic=format-highlighting.